lib/provider/boku.py

```diff
@@ -33,6 +33,10 @@ def create(self, request):
         cleaned = form.cleaned_data
         transaction = cleaned['param']
 
+        # Check the signature shows it was from Boku.
+        log.info('Checking signature')
+        check_sig(request.DATA)
+
         # Verify this against Boku, this will raise errors if there's
         # an issue.
```

Comment:
while you're in here, I'd amend this comment to explain that it only raises an issue in certain circumstances, not for spoofed requests with valid boku transaction IDs.